### ISSUE TITLE
is_array ignored for primitive types

### ISSUE THREAD

**Issue by warhammerkid** [ruby-grape/grape-swagger-entity#6]
One of my entities has a property that is an array of strings. Looking through the specs for grape-swagger, it appears that this is fully supported in params using `is_array: true` (see [parse_params.rb](https://github.com/ruby-grape/grape-swagger/blob/master/lib/grape-swagger/doc_methods/parse_params.rb#L65)), but when I do that in an entity I don't get an array.

**Comment by dblock** [comment]
Want to share some code that demonstrates the problem?

**Comment by warhammerkid** [comment]
Here's an example entity:

```
module Entity
  class AnEntity < Grape::Entity
    expose :tag_names, documentation: {
      type: 'String',
      desc: 'The tag names',
      is_array: true
    }
  end
end
```

I'd expect this to generate in Swagger as something like this:
```
"AnEntity":{
  "type": "object",
  "properties": {
     "tag_names": {
       "type": "array",
       "description": "The tag names",
       "items": {"type": "string"}
     }
  }
}
```

but what I actually get is this:
```
"AnEntity":{
  "type": "object",
  "properties": {
     "tag_names": {
       "type": "string",
       "description": "The tag names"
     }
  }
}
```

Now I'm using version 0.20.3 of grape-swagger, instead of grape-swagger-entity, so it's possible this case does work. However, at a glance, the code only seems to support `is_array` if the type is another entity.

**Comment by Bugagazavr** [comment]
Fixed in `v0.1.4`
